Tests/AsyncAlgorithmsTests/Performance/TestThroughput.swift

```diff
@@ -24,6 +24,11 @@ final class TestThroughput: XCTestCase {
       chain($0, [].async, [].async)
     }
   }
+  func test_chain4() async {
```

Comment:
I think the names of these tests were intended to be per the number of inputs, so this is a test of chain 2 (but two sides active). @kperryua that was your intent right?

Reply:
Ah I see, if that's the case and confirmed I can make the change.

Reply:
I updated the tests, I also extended `measureSequenceThroughput` to take three parameters to cover the three input sequence for `chain`, `merge` and `zip`